service/src/main/java/bio/terra/policy/service/pao/graph/model/AttributeEvaluator.java

```diff
@@ -22,7 +22,6 @@
  * </ol>
  */
 public class AttributeEvaluator {
-  private static final Logger logger = LoggerFactory.getLogger(AttributeEvaluator.class);
```

Comment:
This was unused; making a change so publishing will be triggered to verify my changes work.